eth/client.go

```diff
@@ -37,13 +36,17 @@ type (
 	}
 )
 
+const (
+	defaultPowDepth = uint64(6)
```

Comment:
Please set to 0. You also don't necessarily need to set a const number type, afaik.